core/plugin_impl_base.h

```diff
@@ -17,14 +17,14 @@ class PluginImplBase {
      * when a system is constructed. This does not mean that the system actually
      * exists and is connected, it might just be an empty dummy system.
      *
-     * Plugins should do initialization steps with other parts of DroneCore
+     * Plugins should do initialization steps with other parts of DronecodeSDK
```

Comment:
"... other parts of Dronecode SDK" - This is a reference to the library, not the class.